<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClimbCap</string>
  </property>
  <property name="windowIcon">
   <iconset resource="terminal.qrc">
    <normaloff>:/images/svg/prise.png</normaloff>:/images/svg/prise.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
   <addaction name="actionCalibration"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/svg/marque-detalonnage.png</normaloff>:/images/svg/marque-detalonnage.png</iconset>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
   <property name="toolTip">
    <string>Calibration capteurs</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
